src/test/java/tests/JsonHelperTest.java

```diff
@@ -0,0 +1,191 @@
+        private boolean testBoolean = false;
+        private String testString = "test";
+        private SecondDummy testSecondDummy = new SecondDummy();
+
+        public int getTestInt() {
+            return testInt;
+        }
+
+        public void setTestInt(int testInt) {
+            this.testInt = testInt;
+        }
+
+        public boolean getTestBoolean() {
+            return testBoolean;
+        }
+
+        public void setTestBoolean(boolean testBoolean) {
+            this.testBoolean = testBoolean;
+        }
+
+        public String getTestString() {
+            return testString;
+        }
+
+        public void setTestString(String testString) {
+            this.testString = testString;
+        }
+
+        public SecondDummy getTestSecondDummy() {
+            return testSecondDummy;
+        }
+
+        public void setTestSecondDummy(SecondDummy testSecondDummy) {
+            this.testSecondDummy = testSecondDummy;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) {
+                return true;
+            } else if (o instanceof Dummy) {
+                Dummy od = (Dummy) o;
+                return this.testInt == od.testInt &&
+                        this.testBoolean == od.testBoolean &&
+                        this.testString.equals(od.testString) &&
+                        this.testSecondDummy.equals(od.testSecondDummy);
+            } else {
+                return false;
+            }
+        }
+
+        @Override
+        public int hashCode() {
+            // Does not support hashcode
+            // This practice is recommended by findbugs
+            assert false;
+            return 0;
+        }
+    }
+
+    /**
+     * Dummy class to test json conversion
+     */
+    private static class SecondDummy {
+        // default values
+        private int[] testIntArr = {0, 1, 2};
+
+        public int[] getTestIntArr() {
+            return testIntArr;
+        }
+
+        public void setTestIntArr(int... testIntArr) {
+            this.testIntArr = testIntArr;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) {
+                return true;
+            } else if (o instanceof SecondDummy) {
+                SecondDummy od = (SecondDummy) o;
+                return Arrays.equals(this.testIntArr, od.testIntArr);
+            } else {
+                return false;
+            }
+        }
+
+        @Override
+        public int hashCode() {
+            // Does not support hashcode
+            // This practice is recommended by findbugs
+            assert false;
+            return 0;
+        }
+    }
+
+    @Test
+    public void fromJson_emptyString_nullObject() {
+        String emptyString = "";
+        Dummy dummy = JsonHelper.fromJsonString(emptyString, Dummy.class);
+        assertEquals(null, dummy);
+    }
+
+    @Test
+    public void fromJson_nullJson_defaultObject() {
```

Comment:
Just wondering why the last part of the test name of this test is different from above. What's `defaultObject`? Since we are asserting `null` as well here.